.jenkins/pipelines/OpenEnclave/releases/publish/Jenkinsfile

```diff
@@ -16,59 +16,42 @@ library "OpenEnclaveJenkinsLibrary@${params.OECI_LIB_VERSION}"
 /**
  * Publish OpenEnclave Ubuntu packages to the Microsoft APT repository
  *
- * @param release_version  The version of the Open Enclave release to install
+ * @param release_version  The version of the Open Enclave release to publish
  * @param os_release       The distribution version without "." (e.g. 1804)
  */
 
 def ubuntuPublishRelease(String release_version, String os_release) {
 
     helpers.WaitForAptLock()
     withCredentials([file(credentialsId: 'azure-repoapi-client', variable: 'repoapiclient'),
-                     string(credentialsId: 'msft-apt-repo-pem', variable: 'repoclientpem'),
-                     string(credentialsId: 'ReleaseAADClientId', variable: 'AADClientId'),
-                     string(credentialsId: 'ReleaseAADResource', variable: 'AADResource'),
-                     string(credentialsId: 'ReleaseAADTenant', variable: 'AADTenant'),
-                     string(credentialsId: "MicrosoftAptRepositoryUbuntu${os_release}", variable: "Ubuntu${os_release}repoid")]) {
+                     file(credentialsId: "ubuntu-${os_release}-repo-config", variable: 'aptconfig'),
+                     string(credentialsId: 'msft-apt-repo-pem', variable: 'repoclientpem')]) {
         sh '''#!/bin/bash
             set +x
-            echo Ubuntu${os_release}
+            echo Ubuntu $os_release
             sudo apt-get update
             sudo apt-get -y --no-install-recommends install python3-adal jq
 
             if ! apt list --installed | grep azure-repoapi-client; then
-                sudo apt -y install ${repoapiclient}
+                sudo apt -y install $repoapiclient
             fi
 
             # Install repoclient.pem
-            if [[ -f ${WORKSPACE}/repoclient.pem ]]; then
-                rm ${WORKSPACE}/repoclient.pem
+            if [[ -f $WORKSPACE/repoclient.pem ]]; then
+                rm $WORKSPACE/repoclient.pem
             fi
             touch ~/.ssh/repoclient.pem
             chmod 600 ~/.ssh/repoclient.pem
-            echo ${repoclientpem} | awk 'NF {gsub(/- /, "-\\n"); gsub(/ -/, "\\n-"); printf \$0;}' &>> ${WORKSPACE}/repoclient.pem
+            echo "$repoclientpem" >> $WORKSPACE/repoclient.pem
```

Comment:
This is nice as we can avoid the awk. Though are you sure this works? Last time I tried this, the pem was considered invalid due to the bad formatting.